data/csp/ec2/ondemand/sle15/packages.yaml

```diff
@@ -0,0 +1,6 @@
+packages:
+  image:
+    EC2 ondemand:
```

Comment:
do we want to allow spaces in hash keys ?

Reply:
I think not. If the snippet lives in a tree that is already labeled with `ec2` do we need the code to call this out again as in `ec2-on-demand` or would `on-demand` be sufficient? For example for the profile things are not labeled as `azure-profile` or `ec2-profile`.

Reply:
I don't think spaces should be a problem in keys, but I'm happy to get rid of them. I would keep the EC2 reference though, as it reduces the risk of namespace collision. Also, this way you see where the package selection comes from (it even gets added as a comment in the generated config.kiwi).

Reply:
Well it's about consistency and duplication. I am not certain how `ec2-on-demand` is different from `ec2-profile`, `azure-profile` etc. But for the profile it was chosen not to have such a prefix. I don't think we should block this initial PR on the naming conversation but think we should look at this again.

Reply:
Keys need to be unique (lowest level definition wins) so that's why the package lists are namespaced.

Reply:
:+1: no spaces anymore